README.md

```diff
@@ -34,18 +31,28 @@ We're happy to review and approve new filters and actions that help you integrat
 with a feature add or bug fix are difficult to review and may be rejected
 * Use the latest version of WordPress and turn `WP_DEBUG` on
 * If other plugins are installed during testing that might affect behavior, please list those in the PR description
-* Make sure to test against the lowest PHP version supported (see `Requires PHP:` [here](https://github.com/auth0/wp-auth0/blob/master/readme.txt#L5)) 
+* Make sure to test against the lowest PHP version supported (see `Requires PHP:` [here](blob/master/readme.txt#L5)) 
```

Comment:
This won't work. Either full URL or relative from the root like this -> `readme.txt`. Note using "relative" won't work for different branches than the default one.